test/test.cc

```diff
@@ -309,6 +309,7 @@ boost::property_tree::ptree make_config(const std::string& path_prefix,
         "max_radius": 200,
         "max_reachability": 100,
         "max_timedep_distance": 500000,
+        "max_distance_disable_hierarchy_culling": 0,
```

Comment:
We _could_ remove this kind of change now (since we're also defaulting to `0.f` in the code, not only the config), but might also keep it. I don't have strong feelings around it, just commenting:)